Cargo.toml

```diff
@@ -282,6 +282,10 @@ cfg-if = { version = "1.0.0", default-features = false }
 tonic = { version = "0.5", optional = true, default-features = false, features = ["transport", "codegen", "prost", "tls"] }
 data-encoding = { version = "2.2", default-features = false, features = ["std"], optional = true }
 trust-dns-proto = { version = "0.20", features = ["dnssec"], optional = true }
+mac_address = { version = "1.1.2", default-features = false, optional = true }
+global_counter = { version = "0.2.2", default-features = false, optional = true }
+strum = { version = "0.21.0", default-features = false, optional = true }
+strum_macros = { version = "0.21.0", default-features = false, optional = true }
```

Comment:
Unless we feel confident that these two crates would have broad application in the project I vote not to include them. It looks like the two features being used are possible to replicate briefly without expanding the maintenance burden of the project, require more expertise of the code reader.

Reply:
Agree, removed it.